src/query-builder/update-result.ts

```diff
@@ -1,15 +1,24 @@
+import { setEnumerableProperties } from '../util/object-utils.js'
+
 export class UpdateResult {
   readonly #numUpdatedRows: bigint
+  readonly #numChangedRows: bigint
 
-  constructor(numUpdatedRows: bigint) {
+  constructor(numUpdatedRows: bigint, numChangedRows: bigint) {
     this.#numUpdatedRows = numUpdatedRows
+    this.#numChangedRows = numChangedRows
   }
 
   get numUpdatedRows(): bigint {
     return this.#numUpdatedRows
   }
+
+  /**
+   * MySQL-only
+   */
+  get numChangedRows(): bigint {
+    return this.#numChangedRows
+  }
 }
 
-Object.defineProperty(UpdateResult.prototype, 'numUpdatedRows', {
-  enumerable: true,
-})
+setEnumerableProperties(UpdateResult, 'numUpdatedRows', 'numChangedRows')
```

Comment:
The name of this functions sounds like

```
set the value of these enumerable properties
```

A better name would be `makePropertiesEnumerable`